YoutubeExplode.Tests/ChannelSpecs.cs

```diff
@@ -79,6 +80,7 @@ public async Task I_can_get_videos_uploaded_by_a_YouTube_channel()
 
             // Assert
             videos.Should().HaveCountGreaterOrEqualTo(80);
+            videos.Select(v => v.ChannelId.Value.Should().Be("UCEnBXANsKmyj2r9xVyKoDiQ"));
```

Comment:
There should be a way to assert more directly like so:

`videos.Select(v => v.ChannelId.Value).Should().AllBe("UCEnBXANsKmyj2r9xVyKoDiQ")` (maybe some other method, I don't remember the name)

That would provide better assertion error messages.

Reply:
Right now i found the [IEnumerable\<string> All()](https://docs.microsoft.com/en-us/dotnet/api/system.linq.enumerable.all?view=netcore-3.1) method:

```c#
videos.Select(v => v.ChannelId.Value).All(id => id == "UCEnBXANsKmyj2r9xVyKoDiQ").Should().BeTrue();
```

How to search for alternative implementations?

Reply:
I'm talking about FluentAssertions, not LINQ.

Just checked, it should be 

```csharp
videos.Select(v => v.ChannelId.Value).Should().AllBeEquivalentTo("UCEnBXANsKmyj2r9xVyKoDiQ");
```

Reply:
> 'StringCollectionAssertions' does not contain a definition for 'AllBeEquivalentTo' and no accessible extension method 'AllBeEquivalentTo' accepting a first argument of type 'StringCollectionAssertions' could be found (are you missing a using directive or an assembly reference?) [YoutubeExplode.Tests]

[I think AllBeEquivalentTo](https://github.com/fluentassertions/fluentassertions/blob/3f7078a5dd2134bbc9dd3a9a1817ac0d239c9a52/Src/FluentAssertions/Collections/GenericCollectionAssertions.cs#L550) accepts `IEnumerable<string>` (in this scenario) instead of simple `string`.

Reply:
Hmm weird. Okay, leave it as it is then.